test/cases/api.sh

```diff
@@ -953,4 +953,30 @@ curl \
         --header "x-rh-identity: $INVALIDAUTHSTRING" \
         http://localhost:443/api/composer/v1/version)" = "404" ]
 
+#
+# Make sure that a request that makes the dnf-json crash returns a 500 error
+#
+[ "$(curl \
+    --silent \
+    --output /dev/null \
+    --write-out '%{http_code}' \
+    -k --key /etc/osbuild-composer/client-key.pem \
```

Comment:
At this point osbuild-composer has been restarted to work with the rh-identity header (see line 933 and up). So https and --key/--cert won't work here. Either move above line 919 or change the request here to `http` with `--x-rh-identity: "$VALIDAUTHSTRING"`.

The header is something that gets added when composer is running behind the CRC gateway (which handles auth for us).